test/unit/autocomplete.ts

```diff
@@ -67,6 +67,20 @@ suite('Autocomplete', ({ expect, spy, stub }) => {
     });
   });
 
+  describe('onUnmount()', () => {
+    it('should remove flux event listeners', () => {
+      const off = spy();
+      autocomplete.flux = <any>{ off };
+
+      autocomplete.onUnmount();
+
+      expect(off).to.have.callCount(3)
+        .and.calledWith('sayt:activate_next', autocomplete.activateNext)
```

Comment:
Use `calledWithExactly` here, as `calledWith` will return true if it was called with these arguments, regardless of if there were any more (eg `flux.off('sayt:activate_next', autocomplete.activateNext, 4)` would pass with `calledWith`), `calledWithExactly` just checks that there are no other arguments.